client/components/data/domain-management/email/index.jsx

```diff
@@ -85,3 +94,29 @@ module.exports = React.createClass( {
 		);
 	}
 } );
+
+export default connect(
+	( state, ownProps ) => {
+		let usersGetter;
+		if ( ownProps.selectedDomainName ) {
+			usersGetter = partial( getByDomain, state, ownProps.selectedDomainName );
+		} else {
+			usersGetter = partial( getBySite, state, ownProps.sites.getSelectedSite().ID );
+		}
+		return {
+			googleAppsUsers: usersGetter(),
+			googleAppsUsersLoaded: getLoaded( state )
+		}
+	},
+	( dispatch, ownProps ) => {
+		let usersFetcher;
+		if ( ownProps.selectedDomainName ) {
+			usersFetcher = partial( fetchByDomain, ownProps.selectedDomainName );
+		} else {
+			usersFetcher = partial( fetchBySiteId, ownProps.sites.getSelectedSite().ID );
+		}
+		return {
+			fetch: () => dispatch( usersFetcher() )
```

Comment:
Not sure if this one is necessary either, since we don't pass any other args here to usersFetcher.